src/WebAPI/ConfigureJwtBearerOptions.cs

```diff
@@ -1,28 +0,0 @@
-﻿using Microsoft.AspNetCore.Authentication.JwtBearer;
-using Microsoft.Extensions.Options;
-
-namespace WebAPI;
-
-public class ConfigureJwtBearerOptions : IPostConfigureOptions<JwtBearerOptions>
-{
-    public void PostConfigure(string name, JwtBearerOptions options)
-    {
-        var originalOnMessageReceived = options.Events.OnMessageReceived;
-        options.Events.OnMessageReceived = async context =>
-        {
-            await originalOnMessageReceived(context);
-
-            if (string.IsNullOrEmpty(context.Token))
-            {
-                var accessToken = context.Request.Query["access_token"];
-                var path = context.HttpContext.Request.Path;
-
-                if (!string.IsNullOrEmpty(accessToken) &&
-                    path.StartsWithSegments("/ruleshub"))
-                {
-                    context.Token = accessToken;
-                }
```

Comment:
Sure you don't need this stuff for SignalR auth?

Reply:
Yes, this is a duplicate of the code in https://github.com/SSWConsulting/SSW.Rules.GPT/blob/main/src/WebAPI/Program.cs and was unused.